hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/MiscCryptoSuite.java

```diff
@@ -84,14 +80,7 @@ final Stream<DynamicTest> unsupportedAndUnauthorizedTransactionsAreNotThrottled(
 
     @HapiTest
     final Stream<DynamicTest> verifyUnsupportedOps() {
-        return hapiTest(
-                getClaimNotSupported(),
-                getStakersNotSupported(),
-                getFastRecordNotSupported(),
-                getBySolidityIdNotSupported(),
-                getExecutionTimeNotSupported(),
-                getTokenNftInfosNotSupported(),
-                getAccountNftInfosNotSupported());
+        return hapiTest(getClaimNotSupported(), getBySolidityIdNotSupported(), getExecutionTimeNotSupported());
```

Comment:
Unsure why some of these were removed without replacement?

Reply:
This is verifying that certain queries are not supported.
The removed queries were removed from the protobuf, and _cannot_ be called; so we cannot test that they are not supported because the query doesn't exist.